### DISCUSSION TITLE
Welcome to openv Discussions!

### DISCUSSION THREAD

**Discussion by speters** [openv/openv#511] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Help improving the projects

## 👋 Willkommen!
  Wir nutzen die Diskussionen als einen Ort, um mit anderen in Kontakt zu treten. Wir hoffen, dass du:
  * Fragen stellst,
  * Ideen teilst.
  * Dich mit anderen Community-Mitgliedern austauschst,
  * Andere willkommen heißt und aufgeschlossen bist -  denke daran, dass dies eine Gemeinschaft ist, die wir
  gemeinsam aufbauen 💪,
  * Hilfst, die Projekte weiter zu bringen-

**Comment by mario3456** [comment]
Hallo,
bin sicher etwas spät dran, habe aber jetzt ein openv Projekt mittels esp8266 und optolink an einer Wärmepumpe Vitocal 300 BJ 2010 realisiert.
Die Vorarbeit hier ist super und es war schnell zum Laufen zu bekommen. Ich will eigentlich nur Daten auslesen, die wichtigen Temperaturen habe ich im Zugriff. Was mir fehlt sind die 2 stelligen Fehlermeldungen "D5" oder ähnliches. Weiß jemand, wie man an die ran kommt? Das würde das Projekt vervollkommnen.
Schönen Abend und Danke schonmal.

**Comment by speters** [comment]
Bei meiner `VScotHO1` sind ab Adresse 0x7590 zwanzig Einträge à 9 Bytes für die Fehlerhistorie vorgesehen.
Innerhalb des Eintrages sind dann ein Byte für den Fehlertyp und die anderen für das Fehlerdatum im geräteüblichen BCD Format.
Ich könnte mir vorstellen, dass das bei deiner Anlage _ähnlich_ ist. Einfach mal in dem Adressraum schauen, ob das was nach Fehlerhistorie aussieht.

Eine Übersicht über Fehlercodes ist u.a. im [Wärmeerzeuger Fehlertexte PDF](https://connectivity.viessmann.com/content/dam/vi-micro/CONNECTIVITY/Vitogate/Vitogate-200/7542150-KNX/Fehlercodes/Fehlercodes_fuer_Gas-und_Oelgeraete.pdf/_jcr_content/renditions/original.media_file.download_attachment.file/Fehlercodes_fuer_Gas-und_Oelgeraete.pdf) zu finden.

**Comment by mario3456** [comment]
Danke für die Antwort, ich suche mal um die Adresse drum herum.

Die Fehlercodes passen für die Wärmepumpe nur teilweise, ist aber nicht schlimm, habe die in der Inbetriebnahmeanleitung.

Von meinem iPhone gesendet

**Comment by mario3456** [comment]
Nochmal vielen Dank.
Habe alles gefunden. Ab 0x0700 stehen die Werte bei mir.

Gruß, Mario

Von meinem iPhone gesendet
